Cannot use SQLite database

Migration to database fails on initial setup.

**To Reproduce**
Steps to reproduce the behavior:
1. Run docker
```
docker run \
    --name photoview \
    --detach \
    --publish 127.0.0.1:3080:80 \
    --restart=always \
    -e PHOTOVIEW_DATABASE_DRIVER=sqlite \
    -e PHOTOVIEW_SQLITE_PATH=/app/data/photoview.db \
    -e PHOTOVIEW_LISTEN_IP=localhost \
    -e PHOTOVIEW_LISTEN_PORT=80 \
    -e PHOTOVIEW_MEDIA_CACHE=/app/cache \
    --volume /opt/photoview/data:/app/data \
    --volume /opt/photoview/cache:/app/cache \
    --volume /opt/photoview/photos:/photos:ro \
    viktorstrate/photoview:2
```

docker logs:
```
2022/11/01 18:10:35 Opening SQLITE database: /app/data/photoview.db?cache=shared&mode=rwc
2022/11/01 18:10:38 Migrating `media_exif.flash` from string to int

2022/11/01 18:10:38 /app/database/migration_exif.go:122 no such table: information_schema.columns
[0.192ms] [rows:0] SELECT data_type FROM information_schema.columns WHERE table_name = 'media_exif' AND column_name = 'flash';
2022/11/01 18:10:38 Failed to run exif fields migration: migrating `media_exif.flash` failed: read data_type of column media_exif.flash: no such table: information_schema.columns
2022/11/01 18:10:38 Initializing scanner queue with 1 workers
2022/11/01 18:10:38 Periodic scan interval changed: disabled
2022/11/01 18:10:38 Scan interval runner: Waiting for signal
2022/11/01 18:10:38 Scan interval runner: New ticker detected
2022/11/01 18:10:38 Scan interval runner: Waiting for signal
2022/11/01 18:10:38 Queue waiting
2022/11/01 18:10:39 Found executable worker: darktable (this is darktable-cli 3.8.1)
2022/11/01 18:10:39 Found executable worker: ffmpeg (ffmpeg version 5.0.1-3+b1 Copyright (c) 2000-2022 the FFmpeg developers)
2022/11/01 18:10:39 Found exiftool
2022/11/01 18:10:39 Initializing face detector
2022/11/01 18:10:40 Could not initialize face detector: initialize facedetect recognizer: Unable to open data/models/shape_predictor_5_face_landmarks.dat for reading.
panic: Could not initialize face detector: initialize facedetect recognizer: Unable to open data/models/shape_predictor_5_face_landmarks.dat for reading.


goroutine 1 [running]:
log.Panicf({0xe66c8c?, 0x147ce20?}, {0xc000139f20?, 0x0?, 0xc000139f70?})
	/usr/lib/go-1.18/src/log/log.go:392 +0x67
main.main()
	/app/server.go:62 +0x26d
```

You're use `information_schema.columns`, but SQLite does not have this.

--- a/api/database/migration_exif.go
+++ b/api/database/migration_exif.go
@@ -22,7 +22,7 @@ func migrate_exif_fields(db *gorm.DB) error {
 		for _, exifCol := range mediaExifColumns {
 			if exifCol.Name() == "exposure" {
 				switch exifCol.DatabaseTypeName() {
-				case "double", "numeric", "real", "bigint":
+				case "double", "numeric", "real", "bigint", "integer":
 					// correct type, do nothing
 				default:
 					// do migration
@@ -34,7 +34,7 @@ func migrate_exif_fields(db *gorm.DB) error {
 
 			if exifCol.Name() == "flash" {
 				switch exifCol.DatabaseTypeName() {
-				case "double", "numeric", "real", "bigint":
+				case "double", "numeric", "real", "bigint", "integer":
 					// correct type, do nothing
 				default:
 					// do migration

